dotCMS/src/main/java/com/dotcms/util/CollectionsUtils.java

```diff
@@ -719,4 +716,42 @@
 		return defaultValue;
 	}
 
+    /**
+     * Returns a {@code Collector} that accumulates the input elements into a
+     * new {@link ImmutableList}.
+     *
+     * @param <T> the type of the input elements
+     * @return a {@code Collector} which collects all the input elements into immutable list
+     */
+	public static <T> Collector<T, ImmutableList.Builder<T>, ImmutableList<T>> toImmutableList () {
+	    return new ImmutableListCollector<>();
+    }
+
+	private static class ImmutableListCollector<T> implements Collector<T, ImmutableList.Builder<T>, ImmutableList<T>> {
+        @Override
+        public Supplier<ImmutableList.Builder<T>> supplier() {
+            return ImmutableList.Builder::new;
+        }
+
+        @Override
+        public BiConsumer<ImmutableList.Builder<T>, T> accumulator() {
+            return (b, e) -> b.add(e);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Specify a type for: 'b', 'e' [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2211)